src/ol/format/filter/logicalnary.js

```diff
@@ -0,0 +1,28 @@
+goog.provide('ol.format.filter.LogicalNary');
```

Comment:
This leaves `ol.format.filter.Logical` as a pretty useless "abstract" class - with only one implementation and nothing using the type.  What do you think about removing that?